src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo';
+import '../../App.scss';
+import { Post } from '../../types/post';
+import React from 'react';
+
+type TodoInfoProps = {
+  todo: Post;
+};
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo }) => {
```

Comment:
[CODE STYLE] - Avoid destructuring props in the function signature. It might be harder to understand what props the component expects. Instead, you can destructure them inside the function body. This way, it will be easier to see what props the component expects at a glance.